quantecon/markov/random.py

```diff
@@ -218,7 +219,8 @@ def random_discrete_dp(num_states, num_actions, beta=None,
     ddp = DiscreteDP(R, Q, beta, s_indices, a_indices)
     return ddp
 
-
+#TODO: Should this use (nopython=True) flag?
```

Comment:
In a (near) future PR, I would like to make this function `_sa_indices` public. One possibility is to move it to a new file `markov/utilities.py`. Another is to put it to `gridtools.py`, with an appropriate name?